src/scripts/uo/extract-uo-module.sh

```diff
@@ -0,0 +1,22 @@
+# Download UO release from 2023-05-25
+curl -L https://data.bioontology.org/ontologies/UO/submissions/219/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb > uo-full-download.owl
+# Extract the terms we want with hierarchy
+robot merge --input uo-full-download.owl extract --method MIREOT --lower-terms uo-w-hierarchy.txt --intermediates all --output uo-module-temp.owl
+# add xmlns:obo="http://purl.obolibrary.org/obo/"
+sed -i 's/xmlns:owl/xmlns:obo="http:\/\/purl.obolibrary.org\/obo\/"\
+     xmlns:owl/' uo-module-temp.owl
+# Replace "<rdfs:comment>" with "obo:IAO_0000115" 
```

Comment:
Since this is an actual change to the raw data, I am wondering whether this is the right place or rather oeo-import-edits? Not sure, though.

Reply:
After reading the module description in the wiki again, the edits-module shall contain our axioms. Thus, I think it can stay as is. 👍